### ISSUE TITLE
'State' attribute incorrectly added to XLIFF 2.0 files during syncing

### ISSUE THREAD

**Issue by antpyykk** [rvanbekkum/vsc-xliff-sync#57]
Hi! A minor issue found out during testing.

---

**Version:** 0.5.1

**Steps to reproduce**

1. Add two new XLIFF files - one base file & one translation file
1. Update the base file with a new unit not present in the translation file
1. Run XLIFF sync

**Expected result**
New unit added to translation file, with target element set as an empty element

```xml
<unit id="sample_unit">
	<segment state="initial">
		<source>Not assigned</source>
		<target/>
	</segment>
</unit>
```
 
**Actual result**
Unit added but with an additional incorrect state attribute added to target element.
Unlike in version 1.2, state element can only be added segment elements as per [XLIFF 2.0 core specification](http://docs.oasis-open.org/xliff/xliff-core/v2.0/xliff-core-v2.0.html#state)

```xml
<unit id="sample_unit">
	<segment state="initial">
		<source>Not assigned</source>
		<target state="initial"/>
	</segment>
</unit>
```

---

Once again, thanks for the great work on this extension!

**Comment by rvanbekkum** [comment]
Hi @antpyykk,
Thank you for filing this issue! I will pick this up for the next release. 🙂

**Comment by rvanbekkum** [comment]
Changes for this are shipped in v0.7.0!
